python/ray/data/context.py

```diff
@@ -66,7 +66,7 @@
 DEFAULT_USE_POLARS = False
 
 # Whether to estimate in-memory decoding data size for data source.
-DEFAULT_DECODING_SIZE_ESTIMATION_ENABLED = True
+DEFAULT_DECODING_SIZE_ESTIMATION_ENABLED = False
```

Comment:
Thanks @scv119 for flagging the issue. FYI this is the code disabling sampling logic completely. So below ratio + strategy change is non-effect given this change.